docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -68,11 +68,14 @@ provided through our Web App or you can use our Python package and build a scrip
       **Create a Dataset**
 
       This recipe requires that you already have a dataset in the Lightly Platform
-      configured to use the data in your AWS S3 bucket. Create such a dataset in 2 steps:
+      configured to use the data in your cloud bucket. Create such a dataset in 
+      two steps:
 
       1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly.
-         Make sure that you choose the input type `Images` or `Videos` correctly, depending on the type of files in your cloud storage bucket.
-      2. Edit your dataset, select the storage source as your datasource and fill out the form.
+         Make sure that you choose the input type `Images` or `Videos` correctly, 
+         depending on the type of files in your cloud storage bucket.
+      2. Edit your dataset, select the storage source as your datasource and fill 
```

Comment:
select the storage source as -> select your cloud storage provider as your datasource